### PR TITLE
Updated the error generator script to produce python enum of errors

### PR DESCRIPTION
**Pull request by dimakis**

### CODE REVIEW

File: scripts/errors/generate-errors.js [hunk 1]
```diff
@@ -14,29 +14,19 @@ for (api in apis) {
...
-    })
-    \`\`\`
-*/
-export const APIErrorCodes = {
+  stringBuffer = 
+`from enum import Enum 
+
+#   ${api} error codes
+
+class APIErrorCodes(Enum):
```
Comment (dimakis) [inline scripts/errors/generate-errors.js:22]: This creates a python Enum. from reading an enum appears to be the preferred structure to hold errors as there are no consts in python. 

As it stands though this enum doesn't appear to be used. I'm not sure how the SDK incorporates them. 

any knowledge @wtrocki ?

Comment (wtrocki) [reply scripts/errors/generate-errors.js:22]: Yes. All we need is to import it and use it's value. 
See: 
https://github.com/redhat-developer/app-services-sdk-go/tree/main/registrymgmt#error-handling

**Comment by wtrocki** [comment]
We could add some docs into docs folder later explaining error handling.

**Comment by dimakis** [comment]
> We could add some docs into docs folder later explaining error handling.

yes, this is a great shout. I'm not 💯 on the error handling myself yet, I was going to play around with it a little more to see exactly how it works then doc it

**Pull request merged**